pkg/tfvars/openstack/openstack.go

```diff
@@ -33,12 +33,16 @@ func TFVars(
 	bootstrapIgn string,
 ) ([]byte, error) {
 	var (
-		cloud          = installConfig.Config.Platform.OpenStack.Cloud
-		mastermpool    = installConfig.Config.ControlPlane.Platform.OpenStack
-		defaultmpool   = installConfig.Config.OpenStack.DefaultMachinePlatform
-		machinesSubnet = installConfig.Config.Platform.OpenStack.MachinesSubnet
+		cloud        = installConfig.Config.Platform.OpenStack.Cloud
+		mastermpool  = installConfig.Config.ControlPlane.Platform.OpenStack
+		defaultmpool = installConfig.Config.OpenStack.DefaultMachinePlatform
 	)
 
+	networkClient, err := clientconfig.NewServiceClient("network", openstackdefaults.DefaultClientOpts(cloud))
+	if err != nil {
+		return nil, fmt.Errorf("failed to build an OpenStack service client: %w", err)
+	}
```

Comment:
We use `networkClient` to resolve some missing information, like machinesSubnet's network. Since it's used more than once, I thought I'd instantiate the client once. That also makes more evident where we're using Gophercloud in function calls.